src/background/main.ts

```diff
@@ -1,5 +1,6 @@
 import { addExtensionsListeners } from '@/helpers/extensions';
-import { cleanProxyListeners, initProxyListeners } from '@/helpers/socksProxy';
+import {} from '@/helpers/socksProxy';
```

Comment:
You're not importing anything from socksProxy..?

Reply:
Fixed.